tests/data/transcription/est00.txt

```diff
@@ -0,0 +1,5 @@
+0.120   0.290   225.000
```

Comment:
Oops, I read that you head the ground truth and algorithm output but not the scores.  If you have ground truth and (runnable) code, and some transcription algorithm of some kind, you could use said transcription algorithm to generate estimated annotations and then use the code to create the regression test scores.